cfgmgr/vrfmgr.cpp

```diff
@@ -67,6 +67,13 @@ VrfMgr::VrfMgr(DBConnector *cfgDb, DBConnector *appDb, DBConnector *stateDb, con
                 }
                 else
                 {
+                    // No deletion of mgmt table from kernel
+                    if (vrfName.compare("mgmt") ==0) {
```

Comment:
You may also want to check for `management `- Refer https://github.com/Azure/sonic-utilities/blob/master/config/main.py#L1814

Reply:
In the click, user may use either "config vrf add mgmt" or "config vrf add management" to configure management VRF.
In vrfmgr.cpp, inside  VrfMgr::VrfMgr(...),  vrf names are derived from the output of command "ip -d link show type vrf"
 34     cmd << IP_CMD << " -d link show type vrf";
 35     EXEC_WITH_ERROR_THROW(cmd.str(), res);

I think the name for mgmt vrf is "mgmt" only in the kernel.

root@sonic:~# ip -d link show type vrf
...
51: mgmt: <NOARP,MASTER,UP,LOWER_UP> mtu 65536 qdisc noqueue state UP mode DEFAULT group default qlen 1000
    link/ether 2e:6d:23:6f:7a:de brd ff:ff:ff:ff:ff:ff promiscuity 0 minmtu 68 maxmtu 1500
    vrf table 5000 addrgenmode eui64 numtxqueues 1 numrxqueues 1 gso_max_size 65536 gso_max_segs 65535


Thanks,
Bing

Reply:
Agree